packages/react-canvas-core/src/states/SelectionBoxState.ts

```diff
@@ -58,6 +59,11 @@ export class SelectionBoxState extends AbstractDisplacementState {
 				if (rect.containsPoint(model.getPosition())) {
 					model.setSelected(true);
 				}
+			} else if (model instanceof LinkModel) {
```

Comment:
this unfortunately makes react-canvas dependent on react-diagrams. The correct way to implement this at a framework level is to have all positional elements return a bounding box, and then check the bounding box for all elements universally regardless of if they are links or not. Im going to implement this today.